packages/gql/codegen.ts

```diff
@@ -15,11 +16,13 @@ export const SUBGRAPHS = {
       const baseEndpoint = "https://api.thegraph.com/subgraphs/name/bleu-llc";
 
       return {
-        [Network.Mainnet]: `${baseEndpoint}/balancer-pools-metadata`,
+        [Network.Ethereum]: `${baseEndpoint}/balancer-pools-metadata`,
```

Comment:
Can you leave a comment on the top of this file explaining why we chose to use this pattern here? I think it'd be very hard for a newcomer to reason about this "gambi" 😅

Reply:
It's not clear to me which pattern I should do a comment xD sorry

Reply:
Sorry, should've been more explicit here but I think you could also have thought of what would generate [WTFs](https://miro.medium.com/v2/resize:fit:1000/0*HaqsruEqnR5pxQD4) in this file. We're pointing a bunch networks to a Goerli endpoint -- I assume because you didn't want to deal with TS errors right now -- that's a non obvious choice that I think must be documented here.

Reply:
AAA okkk on it